include/misc/util.h

```diff
@@ -27,6 +27,13 @@
 #define POINTER_TO_INT(x)  ((s32_t) (x))
 #define INT_TO_POINTER(x)  ((void *) (x))
 
+#define BITS_PER_LONG	(__CHAR_BIT__) * (__SIZEOF_LONG__)
```

Comment:
Could make this

```c
#define BITS_PER_LONG (__CHAR_BIT__ * __SIZEOF__LONG__)
```

That's what `tools/include/asm-generic/bitsperlong.h` does.

`CHAR_BIT` could come from `limits.h`, to avoid `__CHAR_BIT__`, though it's probably safe to assume that one exists if the other does, and it saves a header.

Reply:
Or more realistically, you could write `8` I guess.~

Reply:
I would vote for `__CHAR_BIT__` instead of hardcoding and avoid
developers wondering about the magic number.

Reply:
Yeah, no worries, wasn't mean seriously, though I wonder if we'll ever run into any other byte sizes.